mol-renderer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
+MAINTAINER Greg Landrum <greg.landrum@gmail.com>
+
+# adapted from continuumio/miniconda3
+RUN apt-get update --fix-missing && apt-get install -y \
+  wget \
+  bzip2 \
+  ca-certificates \
+  libc6
+
+RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
+    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-3.19.0-Linux-x86_64.sh && \
+    /bin/bash /Miniconda3-3.19.0-Linux-x86_64.sh -b -p /opt/conda && \
+    rm Miniconda3-3.19.0-Linux-x86_64.sh
+
+ENV PATH /opt/conda/bin:$PATH
+ENV LANG C
+
+RUN conda config --add channels  https://conda.binstar.org/greglandrum
```

Comment:
It should be the same code underneath, so yes. But I think that would require me to do separate "conda install" commands.

Note that this is pulling a "fresh-off-the-master" version of the RDKit along with the "cairo_nox" package I created, which is a cairo build done without any X (or GTK) requirements.